test/conftest.py

```diff
@@ -0,0 +1,82 @@
+import typing as t
+
+import pytest
+from aiohttp.test_utils import TestClient
+
+from src.app import create_app
+from src import db
+from src.models.user import User
+from src.models.game import Game
+
+
+@pytest.fixture(autouse=True)
+async def set_up():
+    # Put code here
+    yield
+
+
+@pytest.fixture(autouse=True)
+async def therdown(client: TestClient):
+    yield
+    await db.cleanup(client.app['pool'])
+
+
+@pytest.fixture
+async def client(aiohttp_client) -> TestClient:
+    client = await aiohttp_client(create_app())
+    yield client
+    await client.app['pool'].close()
+
+
+@pytest.fixture
+def user_object() -> User:
+    return User(
+        {
+            'username': 'test_user',
+            'password': '12345'
+        }
+    )
+
+
+@pytest.fixture
+async def game_object(client: TestClient, test_user: User) -> Game:
+    return await db.create_game(
+        client.app['pool'],
+        await db.get_user(client.app['pool'], test_user)
+    )
+
+
+@pytest.fixture
+async def test_user(client: TestClient, user_object: User):
+    await db.create_user(
```

Comment:
How about, to return id from create_user function, set it to the user_object, and then, return it.
Benefits
1. You don't need get user db function anymore
2. You can use logged_user/test_user directly, without any calls.

use case: 
* https://github.com/qerdcv/ttt-online/pull/20/files#diff-7dae78a00d8d69422a3bfbde96c7c5d794eede504a0df763853896a1ba1b5ff2R80
* https://github.com/qerdcv/ttt-online/pull/20/files#diff-7dae78a00d8d69422a3bfbde96c7c5d794eede504a0df763853896a1ba1b5ff2R45
* https://github.com/qerdcv/ttt-online/pull/20/files#diff-3603f8223bd7aa4b9d9e9f093cdf2e730f866e442698dc6fa47ce0ad0cc4cfbdR13
* https://github.com/qerdcv/ttt-online/pull/20/files#diff-3365c0d5a884e1773551bc67b04e473c1f3886f924b1de955b4caf38922f76c6R31
and e.t.c.